### ISSUE TITLE
OnClickListener for ImageButton in ExpandableTextView

### ISSUE THREAD

**Issue by squeeish** [Manabu-GT/ExpandableTextView#20]
I am only able to expand the text by tapping on the text itself; nothing happens if I tap the ImageButton. Is this by design?

**Comment by Manabu-GT** [comment]
No, the text should also expand when the ImageButton is tapped.
Are you possibly setting the clicklistener by yourself on the ImageButton?
If you can share the project which reproduces the issue, I can take a look.

**Comment by pdavid0** [comment]
I have the issue too, just implemented the sample in a recyclerview .

**Comment by barykaed** [comment]
yup.. this is quite annoying. is there no fix yet?
